index.js

```diff
@@ -232,10 +236,10 @@ async function HandleMessage(message, sender, respond) {
   cmd = cmd.toLowerCase();
  
   // removes the bang at the beginning
-  const command = cmd.slice(1)
-  if (Object.keys(dummyMapping).includes(command)) {
-    respond(dummyMapping[command](sender, args))
-  }
+  let command = cmd.slice(1)
+  const fallback = () => `!${command} is not a valid command`
+  const func = dummyMapping?.[command] ?? dummyMapping?.[aliases?.[command]] ?? fallback
+  respond(func(sender,args))
```

Comment:
I think it would be best to still pass the `respond` function to the individual command function. I guess it is just taste if `respond` is passed or if a string is returned, but at least for the test cases it is more traceable where the response was generated, because calling the respond function will generate a stack trace that can be printed in case the test fails, and when returning a string from the function that entry in the stack trace is missing. I also think commands then also have more control over maybe not even emitting text or emitting multiple chat messages.

Reply:
I was thinking of another use case with this approach:
The response function could also have have optional further arguments to indicate using a twitch chat command, or an announcement message or a reply to a message in chat etc., having this optional argument makes it easier to escape data properly when sending messages to chat.